packages/releaseLib/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@coveo/semantic-monorepo-tools",
```

Comment:
In slack you linked this : https://github.com/coveo/semantic-monorepo-tools

What's the right one ? Will you delete the other repo, or is this package the "correct" one ? Kinda confused here.

Reply:
I will delete the local package and release the repo I mentioned, that's what I meant by "I copied it in the repo to test it without releasing the whole suite yet." in the description, sorry if it was unclear.

This package I think should help when `coveo/ui-kit` remove lerna, same for the `coveo/admin-ui` I think

Reply:
Update: I will do a test run with the CLI, it'll help iron out the inevitable bug (😢 )
Then, I'll extract it back to the aforementioned repo.